exercises/hello_rubyric/config.yaml

```diff
@@ -0,0 +1,17 @@
+
+_external: true
+title: Hello Rubyric
+description: A simple grading example with Rubyric and Python
+max_submissions: 10
+max_points: 10
+points_to_pass: 6
+min_group_size: 1
+max_group_size: 2
+key: rubyric-python
+lti: Rubyric
+lti_context_id: ''
+lti_resource_link_id: hello-world
+lti_open_in_iframe: true
```

Comment:
Does this work without the iframe as there is the `aplus_get_and_post`?

Reply:
I guess so. I think I got lost in the documentation because there is no mention about these two options being somehow mutually exclusive. Maybe we need to add some more details to the documentation.

Reply:
They shouldn't be exclusive, but if the communication is between A+ and the browser, then the iframe shouldn't be requires. But if it communicates directly with rubyric, then it is.

Reply:
It was stated in the Aplus Manual
![image](https://user-images.githubusercontent.com/10437475/94431483-05e0a100-019e-11eb-95c8-f67a10ed65ff.png)

Reply:
Then we need to fix that part in the aplus-manual too. You could do it in this same pull request, but keep the git commit separate from the other changes.

Reply:
I wasn't sure where you took that screenshot. If it was in this same Rubyric module, then ignore my comment about separating the git commits. That wouldn't be necessary.